assets/scss/_utility-animations.scss

```diff
@@ -0,0 +1,64 @@
+.fade-in-bottom {
```

Comment:
die ganzen Animationen, würde ich in _animations stecken, weil sie unabhängig von der Utility-Animations sein sollten. Der Bezug existiert nicht. Der Utility-Animations ist es egal, würde nicht 2 Orte für globale CSS Animationen aufmachen.